baseplate/context/__init__.py

```diff
@@ -1,9 +1,8 @@
 """Helpers that integrate common client libraries with baseplate's diagnostics.
 
 This package contains modules which integrate various client libraries with
-Baseplate's diagnostics and tracing facilities. When using these helpers,
-trace information and monitoring are automatically collected and sent on for
-you.
+Baseplate's instrumentation. When using them, trace information is passed on
```

Comment:
> When using them,

:no_bell: This is only partially ambiguous, but it doesn't read 100% smoothly. May be worth considering different wording.